addons/account/models/account_move.py

```diff
@@ -362,7 +362,7 @@ def _cleanup_write_orm_values(self, record, vals):
     # ONCHANGE METHODS
     # -------------------------------------------------------------------------
 
-    def _get_accounting_date(self, invoice_date, has_tax):
+    def _get_accounting_date(self, invoice_date, reverse=False):
```

Comment:
cannot be done like this in stable

Reply:
`invoice_date` is really badly named because this method is also used for miscellaneous operations